sample-apps/delivery-customizations-sample-app/web/frontend/components/CustomizationForm.jsx

```diff
@@ -0,0 +1,51 @@
+import { Form, FormLayout, TextField, Button, Badge } from "@shopify/polaris";
+
+export function CustomizationForm({
+  deliveryOptionName,
+  enabled,
+  operation,
+  isNewCustomization = false,
+  onSubmit,
+  onInputChange,
+  loading,
+  disabled,
+  activeStatus,
+  ...props
+}) {
+  const handleSubmit = () => {
+    if (onSubmit) onSubmit({ deliveryOptionName, enabled, operation });
+  };
+
+  const handleInputChange = (value, name) => {
+    if (onInputChange) onInputChange({ value, name });
+  };
+
+  return (
+    <Form onSubmit={handleSubmit} {...props}>
+      <FormLayout>
+        <FormLayout.Group>
+          {activeStatus || isNewCustomization ? (
```

Comment:
What's the difference between `activeStatus` and `enabled`?

Reply:
When I was tophatting, I made my customization inactive through the settings and it still showed as "active" here.

Reply:
Removed the activeStatus, as it was same thing as enabled.